packages/snap/src/polkadot/events/balance/index.ts

```diff
@@ -2,12 +2,26 @@ import {Wallet} from "../../../interfaces";
 import {polkadotEventEmitter} from "../index";
 import {getApi} from "../../api";
 
+let unsubscribe: Record<string, () => void>;
```

Comment:
@mpetrunic not sure if this is the best way, but we need to store these unsubscribe callbacks provided from Polkadot API after subscribing.

Reply:
I guess you cannot pass it on to client?

If you could that would be ideal as it's not really our place to store those